src/Libraries/CoreNodes/Math.cs

```diff
@@ -231,8 +231,8 @@ public static double DegreesToRadians(double degrees)
         /// <summary>
         ///     Finds the absolute value of a number.
         /// </summary>
-        /// <param name="number">A number.</param>
-        /// <returns name="absoluteValue">Absolute value of the number.</returns>
+        /// <param name="number">Number to find absolute value</param>
```

Comment:
`Number to get absolute value from`? think this is more consistent with the pattern we agreed on?

Reply:
adressed in 
16df342